docs/migration/manual-migration.mdx

```diff
@@ -13,69 +13,103 @@ parts, or debug issues in the migration CLI output.
+a truthy value, the resulting component had the responsibility of displaying the
 result of the command with the `InlineAlert` component. In version 3,
 `makeCommandDispatcherWithCommandFeedback` now expects an optional React
 component to display the result of the command.
 
 To cope with this change, you need to review all components using
 `makeCommandDispatcherWithCommandFeedback`:
 
-* if `makeCommandDispatcherWithCommandFeedback` is called with a falsy second
+- if `makeCommandDispatcherWithCommandFeedback` is called with a falsy second
   parameter, you don't have to change anything
-* if `makeCommandDispatcherWithCommandFeedback` is called with a truthy second
+- if `makeCommandDispatcherWithCommandFeedback` is called with a truthy second
   parameter or without the second parameter, you have to adapt your code. To
   have exactly the same behavior as before, the new call should look like:
   ```jsx
-  makeCommandDispatcherWithCommandFeedback({ /* commands definition */ }, ({ success, message }) => {
-    return <InlineAlert appearance={success ? "success" : "error"}>{message}</InlineAlert>;
-  })
+  makeCommandDispatcherWithCommandFeedback(
+    {
+      /* commands definition */
+    },
+    ({ success, message }) => {
+      return (
+        <InlineAlert appearance={success ? "success" : "error"}>
+          {message}
+        </InlineAlert>
+      );
+    }
+  );
   ```
 
 ## Sass Migration
 
-In the latest version of Front-Commerce we converted all global Sass variables to CSS
-variables during the [automated migration](./automated-migration). This allows
-for a more modular approach to theming and a better developer experience.
+In the latest version of Front-Commerce we converted all global Sass variables
+to CSS variables during the [automated migration](./automated-migration). This
+allows for a more modular approach to theming and a better developer experience.
 However, there are still some manual steps that need to be taken to complete the
 migration.
 
-### Media Mixin
+### Breakpoints Deprecations
+
+:::info
+
+After running the codemods on your project, you can easily find these
+deprecations by searching `TODO (Codemod generated)` in your project.
+
+:::
+
+#### `grid-breakpoints` Variable
 
-If you make use of the
+The `grid-breakpoints` variable has been deprecated, If you make use of the
+[`grid-breakpoints`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss#L4-12)
+variable, you will need to manually implement this variable in your project.
+
+#### `breakpoint-min` Function
+
+The `breakpoint-min` function has been deprecated, If you make use of the
+[`media-breakpoint-up`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss#L14-21)
```

Comment:
**issue:** incorrect link and name

**suggestion:** use `breakpoint-min`'s correct one

Reply:
Hmm not sure I follow it is breakpoint-min?

![image](https://github.com/front-commerce/developers.front-commerce.com/assets/39598117/4f2d04d8-fa01-4b57-be1d-d20a0dd4b8f1)

Reply:
The link seems similar to the other section: 
![2023-08-11_13-23](https://github.com/front-commerce/developers.front-commerce.com/assets/75968/20276821-7044-4db1-92f1-2b392e0c774c)

I meant that it very likely was a copy/paste error.